packs/backgrounds/willowshore-urchin.json

```diff
@@ -0,0 +1,142 @@
c-actions.Item.Seasonal Boon (Willowshore Urchin )]</p>\n<hr />\n<p>You can add one of the following two adjustments to any background to further modify your choice to match your vision for your PC.</p>\n<p><strong>Family:</strong> You and one (or more) other PCs are close family members (siblings, parent/child, married, etc.) If you take this adjustment, the other PC(s) who are your family must also take this adjustment. Choose one of the following to replace the skill feat you otherwise would've gained from your background. (You and your sibling need not share the same choice—a sibling who wants to keep their background's skill feat can pick \"Typical\" from the list.)</p>\n<ul>\n<li><strong>Bickering:</strong> You have a long-standing feud with your sibling(s). Whenever your sibling(s) critically fail a significant skill check or attack roll and you witness their failure, you gain a +1 status bonus to your saving throws until the end of your next turn as you secretly (or perhaps not so secretly) delight in their failure. @UUID[Compendium.pf2e.campaign-effects.Item.Effect: Bickering Family]</li>\n<li><strong>Close-Knit:</strong> You and your sibling(s) work well together, and you go all-out to help them when they're badly hurt. When you provide @UUID[Compendium.pf2e.actionspf2e.Item.Aid] or @UUID[Compendium.pf2e.actionspf2e.Item.Administer First Aid] to your sibling(s), the DC is 5 lower than normal.</li>\n<li><strong>Typical:</strong> You have a standard relationship with your sibling(s); don't change your background's skill feat.</li>\n</ul>\n<p><strong>Foundling:</strong> You aren't a native to Willowshore, but you did grow up in the town. You might have adoptive parents, or you could've grown up on the streets. Choose one of the following to replace the bonus feat you otherwise would've gained from your background.</p>\n<ul>\n<li><strong>Bitter:</strong> You appreciate the town of Willowshore but still resent the conditions that resulted in you being abandoned or left in the town. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Intimidating Glare] skill feat.</li>\n<li><strong>Mysterious:</strong> You keep secrets, and very few in Willowshore (if anyone) know your history before you settled in the town, in part because you've taken to learning about Willowshore by watching rather than engaging in conversation. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Streetwise] skill feat.</li>\n<li><strong>Outgoing:</strong> You're grateful for the sense of belonging the townsfolk of Willowshore have extended to you and always do your best to repay that kindness. Once per week when you attempt a skill check to resolve a downtime activity that could benefit Willowshore (several of these activities are detailed in the adventures, and your GM can always make the call if another activity would qualify) and you fail that check, you instead succeed at the check, or if you critically fail at that check, you instead simply fail that check.</li>\n</ul>"
```

Comment:
```suggestion
            "value": "<p>As with any settlement, not all of Willowshore's citizens are born into easy lives. You grew up on the town streets, either as an orphan or hardscrabble youth who was forced at a young age to help provide for your family. You might technically have had a home of your own, but for various reasons, you had, or preferred, to live most of your childhood outside of those walls. You might have run with a gang of kids (some of whom might even be fellow PCs), or you could've been a loner who managed to survive against all odds. Regardless, your childhood was one of forced ingenuity, desperation, and physicality. Hardly a week would've passed without you having to jury-rig a solution to some problem or to simply use your reflexes or brawn to solve a situation quickly before things escalated. While you might have left your old life behind you now that you're all grown up, the time you spent on Willowshore's streets have made you the person you are today.</p>\n<p>You might still be technically homeless, subsisting on the street and never sleeping in the same place twice in a row. You might keep a small shanty or hideout somewhere in town as well. It's also possible that, despite your childhood, you've been fortunate enough to find a family or home, in which case your home could be anywhere in Willowshore.</p>\n<p>During the Reenactment Festival, you likely enjoyed the revelry... perhaps a bit too much! Festivals were always times of excitement in your childhood, and this one is no exception. You were chosen to be an abductee during the Reenactment Festival simply by chance—maybe your name was selected by a blind draw, or perhaps someone dared you to take part, or you could simply have craved the attention.</p>\n<p>Choose two ability boosts. One must be to <strong>Dexterity</strong> or <strong>Strength</strong>, and one is a free ability boost.</p>\n<p>You're trained in the Athletics skill and the Engineering Lore skill. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Charming Liar] feat.</p>\n<p>@UUID[Compendium.pf2e.adventure-specific-actions.Item.Seasonal Boon (Willowshore Urchin)]</p>\n<hr />\n<p>You can add one of the following two adjustments to any background to further modify your choice to match your vision for your PC.</p>\n<p><strong>Family:</strong> You and one (or more) other PCs are close family members (siblings, parent/child, married, etc.) If you take this adjustment, the other PC(s) who are your family must also take this adjustment. Choose one of the following to replace the skill feat you otherwise would've gained from your background. (You and your sibling need not share the same choice—a sibling who wants to keep their background's skill feat can pick \"Typical\" from the list.)</p>\n<ul>\n<li><strong>Bickering:</strong> You have a long-standing feud with your sibling(s). Whenever your sibling(s) critically fail a significant skill check or attack roll and you witness their failure, you gain a +1 status bonus to your saving throws until the end of your next turn as you secretly (or perhaps not so secretly) delight in their failure. @UUID[Compendium.pf2e.campaign-effects.Item.Effect: Bickering Family]</li>\n<li><strong>Close-Knit:</strong> You and your sibling(s) work well together, and you go all-out to help them when they're badly hurt. When you provide @UUID[Compendium.pf2e.actionspf2e.Item.Aid] or @UUID[Compendium.pf2e.actionspf2e.Item.Administer First Aid] to your sibling(s), the DC is 5 lower than normal.</li>\n<li><strong>Typical:</strong> You have a standard relationship with your sibling(s); don't change your background's skill feat.</li>\n</ul>\n<p><strong>Foundling:</strong> You aren't a native to Willowshore, but you did grow up in the town. You might have adoptive parents, or you could've grown up on the streets. Choose one of the following to replace the bonus feat you otherwise would've gained from your background.</p>\n<ul>\n<li><strong>Bitter:</strong> You appreciate the town of Willowshore but still resent the conditions that resulted in you being abandoned or left in the town. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Intimidating Glare] skill feat.</li>\n<li><strong>Mysterious:</strong> You keep secrets, and very few in Willowshore (if anyone) know your history before you settled in the town, in part because you've taken to learning about Willowshore by watching rather than engaging in conversation. You gain the @UUID[Compendium.pf2e.feats-srd.Item.Streetwise] skill feat.</li>\n<li><strong>Outgoing:</strong> You're grateful for the sense of belonging the townsfolk of Willowshore have extended to you and always do your best to repay that kindness. Once per week when you attempt a skill check to resolve a downtime activity that could benefit Willowshore (several of these activities are detailed in the adventures, and your GM can always make the call if another activity would qualify) and you fail that check, you instead succeed at the check, or if you critically fail at that check, you instead simply fail that check.</li>\n</ul>"
```